src/main/target/OMNIBUSF4/target.h

```diff
@@ -73,7 +73,7 @@
 
 #define USE_EXTI
 
-#define USE_ACC
+//#define USE_ACC
```

Comment:
@jflyper: Oops. Took me a lot of digging and a bisect to figure out why #7541 does not work as it should on OMNIBUSF4. :sweat_smile: Added a fix for it to #7541.

Reply:
Thanks @mikeller